jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -800,6 +770,8 @@ protected void doStart() throws Exception
         if (_mimeTypes == null)
             _mimeTypes = new MimeTypes();
 
+        _durableListeners.addAll(getEventListeners());
```

Comment:
I don't understand this. The durable listeners were added to the list/set when we added a listener and we were not starting/started. Now you seem to be adding them to the durable listeners AFTER they've been removed?????